src/store/gameSlice.ts

```diff
@@ -8,7 +8,8 @@ interface initialStateType {
   guessesRemaining: number;
   winner: string;
   loser: string;
-  gameHistory: []
+  gameHistory: [];
+  host: {} | null;
```

Comment:
(referring to the 'host' type): Unlike the last comment, I think this one needs to be fixed in this merge, the last one Rose and I fixed on the front end branch.
So here host can either be a boolean or null. `host: boolean | null` or its just a boolean and we init the state as false? Up to you!